lmfdb/ecnf/main.py

```diff
@@ -751,6 +751,13 @@ def __init__(self):
             name="include_cm",
             label="CM",
             knowl="ec.complex_multiplication")
+        cm_field = TextBox(
+            name="cm_field",
+            label= "%s field %s" % (display_knowl("ec.complex_multiplication", "CM"), display_knowl("nf.discriminant", "discriminant")),
+            example="-4",
+            example_span="-4 or -3,-8",
+            knowl="None"
```

Comment:
You  should be able to just omit this.

Reply:
Sorry, which part can I omit? The knowl="None"?

Reply:
Yep.  I'm not sure why you would need to have knowl set to `"None"`....